Test/ContentPipeline/TextureImporterTests.cs

```diff
@@ -29,51 +29,85 @@ void ImportStandard(string filename)
             Assert.AreEqual(content.Faces[0][0].Height, 64);
             SurfaceFormat format;
             Assert.True(content.Faces[0][0].TryGetFormat(out format));
-            Assert.AreEqual(format, SurfaceFormat.Color);
+            Assert.AreEqual(expectedSurfaceFormat, format);
             // Clean-up the directories it may have produced, ignoring DirectoryNotFound exceptions
             try
             {
                 Directory.Delete(intermediateDirectory, true);
                 Directory.Delete(outputDirectory, true);
             }
-            catch (DirectoryNotFoundException)
-            { }
+            catch(DirectoryNotFoundException)
+            {
+            }
+        }
+
+        [Test]
+        public void ImportBmp( )
+        {
+            ImportStandard("Assets/Textures/LogoOnly_64px.bmp", SurfaceFormat.Color);
+        }
+        [Test]
+        public void ImportBmpRGB555( )
+        {
+            ImportStandard("Assets/Textures/Logo555.bmp", SurfaceFormat.Color);
+        }
+        [Test]
+        public void ImportBmpRGB565( )
+        {
+            ImportStandard("Assets/Textures/Logo565.bmp", SurfaceFormat.Color);
+        }
+        [Test]
+        public void ImportBmp4bits( )
+        {
+            ImportStandard("Assets/Textures/LogoOnly_64px-4bits.bmp", SurfaceFormat.Color);
         }
 
         [Test]
-        public void ImportBmp()
+        public void ImportBmpMonochrome( )
         {
-            ImportStandard("Assets/Textures/LogoOnly_64px.bmp");
+            ImportStandard("Assets/Textures/LogoOnly_64px-monochrome.bmp", SurfaceFormat.Color);
         }
 
         [Test]
-        public void ImportGif()
+        public void ImportGif( )
```

Comment:
More unnecessary reformatting.  Also check the [Coding Guidelines](https://github.com/mono/MonoGame/wiki/Coding-Guidelines) for rules regarding spaces.